ServerCore/Pages/Events/CreateDemo.cshtml.cs

```diff
@@ -355,19 +355,19 @@ public async Task<IActionResult> OnPostAsync()
                 //
                 // Create teams. Can we add players to these?
                 //
-                Team team1 = new Team { Name = "Team Bugs", Event = Event, IsLookingForTeammates = true };
+                Team team1 = new Team { Name = "Team Bugs", Event = Event, IsLookingForTeammates = true, PrimaryContactEmail = "bugs@bugs.bugs" };
```

Comment:
Maybe we should use example.com for the email domains, just so we don't accidentally use addresses that resolve to real email addresses.